sdk/python/packages/flet-core/src/flet_core/app_bar.py

```diff
@@ -191,3 +203,197 @@ def actions(self):
     @actions.setter
     def actions(self, value):
         self.__actions = value if value is not None else []
+
+
+class BottomAppBar(ConstrainedControl):
```

Comment:
Move `BottomAppBar` class to a separate file please.

Reply:
Ok. I didn't want to do this because the files are getting quite a lot.